stone/backend.py

```diff
@@ -292,7 +292,7 @@ def emit_wrapped_text(
 
     @classmethod
     def process_doc(cls, doc, handler):
-        # type: (str, typing.Callable[[str, str], str]) -> str
+        # type: (str, typing.Callable[[str, str], str]) -> typing.Text
```

Comment:
I'm pretty sure this necessary to avoid error because `stone/backend.py` has `from __future__ import … unicode_literals`. (See below.)